### ISSUE TITLE
Is there a way to disable the GUI and add a password for the API service

### ISSUE THREAD

**Issue by taicaile** [caronc/apprise-api#70]
:question: **Question**

Hello,

May I know if the default WEB page can be disabled? just leave the API service only.

And if there is a method to config a password for the API service?

Thanks.

**Comment by caronc** [comment]
> May I know if the default WEB page can be disabled? just leave the API service only.

Yes, this is done by setting up your configuration the way you want, and then in the `settings.py` set `APPRISE_CONFIG_LOCK` environment variable to `yes`.

> And if there is a method to config a password for the API service?

The bet way to do this is to park an NginX server in-front and set it up as a reverse proxy.  It's not too difficult to do.  From here you can configure an http user/pass that forces you to authenticate before accessing the Apprise Server.

All of your `apprise` calls that used to reference `apprise://hostname:8000` just change to `apprise://user:pass@hostname`
